src/spec/features/password_resets_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+feature 'Password reset' do
+  let(:user) { create(:participant, name: 'Hedy Lamarr', email: 'freq-hopper@example.org') }
+
+  scenario 'The home page has a link to the reset password page' do
+    visit root_path
+    assert page.has_link?('Forgot Password?', href: new_password_reset_path)
+  end
+
+  scenario 'The reset password page has the expected content' do
+    visit new_password_reset_path
+    assert page.has_css?('h1', text: 'Reset Password')
+    assert page.has_text?('Please enter your email address below')
+  end
+
+  scenario 'Participant can reset password' do
+    # Event needed until Pull Request #202 is merged into the code base
+    Event.create!(name: 'Event 1', date: Time.now)
+
+    # Enter an invalid email address
+    visit new_password_reset_path
+    fill_in 'email', with: 'not_exist@example.com'
+    click_button 'Reset Password'
+    assert page.has_text?('No participant was found with email address not_exist@example.com')
+
+    # Enter a valid email address
```

Comment:
I'd make each one of these individual example blocks. `it "enters a valid email address" do ` etc...